requirements.txt

```diff
@@ -1,11 +1,11 @@
-Pyro4==4.31
-Twisted==19.7.0
-Whoosh==2.6.0
-argparse==1.2.1
-ijson==2.0
-psutil==2.1.3
-pycryptopp==0.6.0.1206569328141510525648634803928199668821045408958
-serpent==1.7
-defusedxml==0.4.1
-wsgiref==0.1.2
-zope.interface==4.1.2
+Pyro4
+Twisted
+Whoosh
+argparse
+ijson
+psutil
+pycryptopp
+serpent
+defusedxml
+wsgiref
```

Comment:
That dependency is only needed for Python < 3.2. I guess we could remove it, as we are targeting Python 3.10

Reply:
Aye, removed!